dali/operators/reader/coco_reader_op.cc

```diff
@@ -39,6 +39,17 @@ Tensor (`m` * `[x, y, w, h] or `m` * [left, top, right, bottom]`) and labels as
   .AddOptionalArg("ltrb",
       R"code(If true, bboxes are returned as [left, top, right, bottom], else [x, y, width, height].)code",
       false)
+  .AddOptionalArg("masks",
+      R"code(If true, segmentation masks are read and returned as polygons.
+Each mask can be one or more polygons. A polygon is a list of points (2 floats).
+For a given sample, the polygons are represented by two tensors:
+  `masks_meta` -> list of tuples (mask_idx, start_idx, count)
```

Comment:
Maybe we should stick to sparse tensor format - I mean to return 3 tensors:
- values
- start
- count

Reply:
We also need to carry `mask_idx` to know which polygon belongs to which mask